styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: pink;
+}
```

Comment:
- [ ] Please add the .gitignore file to keep the following:

- node_modules
- test.md